components/Chat/SubmitButton.tsx

```
@@ -1,6 +1,6 @@
 import { ArrowUpRightIcon } from "lucide-react";
 
-const SubmitButton = ({ canSubmit }: { canSubmit: boolean }) => {
+const SubmitButton = ({ canSubmit }: { canSubmit: Boolean }) => {
```

Biome

[error] Don't use 'Boolean' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'boolean' instead


(lint/complexity/noBannedTypes)